python/dlr/tf_model.py

```diff
@@ -0,0 +1,143 @@
+import tensorflow as tf
```

Comment:
1. Test file name is `test_tf2.py`. Lets rename `tf_model.py` to `tf2_model.py` too.
2. Should we check that tf version is 2.x?

Reply:
1. done
2. added `assert version.parse(tf.__version__) >= version.parse(TENSORFLOW_VERSION)` to check tensorflow version